src/terra-dev-site/doc/Configuration.b/webpack.config.e.doc.md

```diff
@@ -0,0 +1,41 @@
+# Webpack Config
+The provided webpack config is the entry point for spinning up terra dev site. By either extending or using directly you'll enable Terra dev site config to be automatically created based on either the custom or default site config.
```

Comment:
terra-dev-site.

Reply:
By either extending or using directly ~you'll enable~ **the** terra-dev-site config  ~to be~ will **be** created